js/ui/popover.js

```diff
@@ -69,7 +70,12 @@ const getEventDelay = function(that, optionName) {
 const attachEvent = function(that, name) {
     const target = that.option('target');
     const isSelector = isString(target);
-    const event = getEventName(that, name + 'Event');
+    let event = getEventName(that, name + 'Event');
+
+    if(that.option('shading') && name === 'hide' && that.option('hideEvent')) {
```

Comment:
Should case with `hideEvent=click` be processed here? Seems like the warning in this scenario is excess

Reply:
Fixed